app-header/app-header.html

```diff
@@ -245,13 +255,20 @@
        * @type {number}
        */
       _deltaHeight: 0,
+      
+      /**
+       * The offsetTop of `_topBar`
+       *
+       * @type {number}
+       */
+      _topbarTop: 0,
```

Comment:
Change to `_topBarTop`, or rename to `_primaryTop`, `_primaryElTop`, `_primaryElemTop`, or something similar (and similarly to `_topBar`)?